src/Cafe/CMakeLists.txt

```diff
@@ -1,29 +1,489 @@
+  OS/libs/gx2/GX2_Texture.h
+  OS/libs/gx2/GX2_TilingAperture.cpp
+  OS/libs/h264_avc/H264Dec.cpp
+  OS/libs/h264_avc/h264dec.h
+  OS/libs/h264_avc/parser/H264Parser.cpp
+  OS/libs/h264_avc/parser/H264Parser.h
+  OS/libs/mic/mic.cpp
+  OS/libs/mic/mic.h
+  OS/libs/nlibcurl/nlibcurl.cpp
+  OS/libs/nlibcurl/nlibcurlDebug.hpp
+  OS/libs/nlibcurl/nlibcurl.h
+  OS/libs/nlibnss/nlibnss.cpp
+  OS/libs/nlibnss/nlibnss.h
+  OS/libs/nn_ac/nn_ac.cpp
+  OS/libs/nn_ac/nn_ac.h
+  OS/libs/nn_acp/nn_acp.cpp
+  OS/libs/nn_acp/nn_acp.h
+  OS/libs/nn_act/nn_act.cpp
+  OS/libs/nn_act/nn_act.h
+  OS/libs/nn_aoc/nn_aoc.cpp
+  OS/libs/nn_aoc/nn_aoc.h
+  OS/libs/nn_boss/nn_boss.cpp
+  OS/libs/nn_boss/nn_boss.h
+  OS/libs/nn_ccr/nn_ccr.cpp
+  OS/libs/nn_ccr/nn_ccr.h
+  OS/libs/nn_cmpt/nn_cmpt.cpp
+  OS/libs/nn_cmpt/nn_cmpt.h
+  OS/libs/nn_common.h
+  OS/libs/nn_ec/nn_ec.cpp
+  OS/libs/nn_ec/nn_ec.h
+  OS/libs/nn_fp/nn_fp.cpp
+  OS/libs/nn_fp/nn_fp.h
+  OS/libs/nn_idbe/nn_idbe.cpp
+  OS/libs/nn_idbe/nn_idbe.h
+  OS/libs/nn_ndm/nn_ndm.cpp
+  OS/libs/nn_ndm/nn_ndm.h
+  OS/libs/nn_nfp/AmiiboCrypto.h
+  OS/libs/nn_nfp/nn_nfp.cpp
+  OS/libs/nn_nfp/nn_nfp.h
+  OS/libs/nn_nim/nn_nim.cpp
+  OS/libs/nn_nim/nn_nim.h
+  OS/libs/nn_olv/nn_olv.cpp
+  OS/libs/nn_olv/nn_olv.h
+  OS/libs/nn_pdm/nn_pdm.cpp
+  OS/libs/nn_pdm/nn_pdm.h
+  OS/libs/nn_save/nn_save.cpp
+  OS/libs/nn_save/nn_save.h
+  OS/libs/nn_temp/nn_temp.cpp
+  OS/libs/nn_temp/nn_temp.h
+  OS/libs/nn_uds/nn_uds.cpp
+  OS/libs/nn_uds/nn_uds.h
+  OS/libs/nsyshid/nsyshid.cpp
+  OS/libs/nsyshid/nsyshid.h
+  OS/libs/nsyskbd/nsyskbd.cpp
+  OS/libs/nsyskbd/nsyskbd.h
+  OS/libs/nsysnet/nsysnet.cpp
+  OS/libs/nsysnet/nsysnet.h
+  OS/libs/padscore/padscore.cpp
+  OS/libs/padscore/padscore.h
+  OS/libs/proc_ui/proc_ui.cpp
+  OS/libs/proc_ui/proc_ui.h
+  OS/libs/snd_core/ax_aux.cpp
+  OS/libs/snd_core/ax_exports.cpp
+  OS/libs/snd_core/ax.h
+  OS/libs/snd_core/ax_internal.h
+  OS/libs/snd_core/ax_ist.cpp
+  OS/libs/snd_core/ax_mix.cpp
+  OS/libs/snd_core/ax_multivoice.cpp
+  OS/libs/snd_core/ax_out.cpp
+  OS/libs/snd_core/ax_voice.cpp
+  OS/libs/snd_user/snd_user.cpp
+  OS/libs/snd_user/snd_user.h
+  OS/libs/swkbd/swkbd.cpp
+  OS/libs/swkbd/swkbd.h
+  OS/libs/sysapp/sysapp.cpp
+  OS/libs/sysapp/sysapp.h
+  OS/libs/TCL/TCL.cpp
+  OS/libs/TCL/TCL.h
+  OS/libs/vpad/vpad.cpp
+  OS/libs/vpad/vpad.h
+  OS/libs/zlib125/zlib125.cpp
+  OS/libs/zlib125/zlib125.h
+  OS/RPL/elf.cpp
+  OS/RPL/rpl.cpp
+  OS/RPL/rpl_debug_symbols.cpp
+  OS/RPL/rpl_debug_symbols.h
+  OS/RPL/rpl.h
+  OS/RPL/rpl_structs.h
+  OS/RPL/rpl_symbol_storage.cpp
+  OS/RPL/rpl_symbol_storage.h
+  TitleList/BaseInfo.cpp
+  TitleList/BaseInfo.h
+  TitleList/GameInfo.h
+  TitleList/MetaInfo.cpp
+  TitleList/MetaInfo.h
+  TitleList/ParsedMetaXml.h
+  TitleList/SaveInfo.cpp
+  TitleList/SaveInfo.h
+  TitleList/SaveList.cpp
+  TitleList/SaveList.h
+  TitleList/TitleId.h
+  TitleList/TitleInfo.cpp
+  TitleList/TitleInfo.h
+  TitleList/TitleList.cpp
+  TitleList/TitleList.h
+  )
```

Comment:
If they were using globs it probably means that they don't want to list files explicitly. I know it is considered a bad practise, but I would ask the dev's opinion first, as keeping track of this many files is error prone.

Reply:
Nah there is no deeper meaning. The CMakeLists.txt files were cobbled together when port work started and I never got to clean or optimize them. We should list all the files explicitly.